sample_project/Assets/SampleViewer/Samples/Routing/RouteManager.cs

```diff
@@ -109,12 +123,31 @@ async void Update()
                     else
                     {
                         routing = true;
-                        StartCoroutine(DrawRoute(results));                       
+                        StartCoroutine(DrawRoute(results));
                     }
                 }
             }
         }
+    }
 
+    void Start()
+    {
+        // We need HPRoot for the HitToGeoPosition Method
+        hpRoot = FindObjectOfType<HPRoot>();
+
+        // We need this ArcGISMapComponent for the FromCartesianPosition Method
+        // defined on the ArcGISMapComponent.View
+        arcGISMapComponent = FindObjectOfType<ArcGISMapComponent>();
+
+        animator = GameObject.Find("InfoMenu").GetComponent<Animator>();
+
+        lineRenderer = Route.GetComponent<LineRenderer>();
+
+        lastRootPosition = arcGISMapComponent.GetComponent<HPRoot>().RootUniversePosition;
+    }
+
+    void Update()
+    {
         RebaseRoute();
```

Comment:
```suggestion

```

Reply:
Removing `RebaseRoute()` from `update` so that we don't need to do this every frame but only after clicking.